validator/src/bin/solana-test-validator.rs

```diff
@@ -187,6 +187,13 @@ fn main() {
                 .takes_value(false)
                 .help("Disable the just-in-time compiler and instead use the interpreter for BPF. Windows always disables JIT."),
         )
+        .arg(
+            Arg::with_name("ticks_per_slot")
+                .long("ticks-per-slot")
+                .value_name("TICKS")
+                .takes_value(true)
```

Comment:
Can you please add a `.validator(is_parsable::<u64>)` here, for better user error reporting.

`is_parsable` is in this module: `solana_clap_utils::input_parsers`

Reply:
thanks for the feedback, i've made the adjustments.